source/app/ImmuneMLApp.py

```diff
@@ -38,11 +94,11 @@ def run(self):
 
         self.set_cache()
 
-        print(f"{datetime.datetime.now()}: ImmuneML: parsing the specification...\n", flush=True)
+        print(f"{datetime.datetime.now().strftime(_DATE_FMT)}: ImmuneML: parsing the specification...\n", flush=True)
```

Comment:
I have no preference between the old and the new date formatting, but important is that these print statements are printed to the log file, and we are using the same (old) formatting all over immuneML (e.g., when entering and exiting encoding, reports, ML method training, etc..). 
If we really want to change the format, I think we should find a better way to consistently retrieve the correct string from any place in immuneML, and have the _DATE_FMT constant for example in the class Constants instead of a global variable.

Reply:
agree, but way too much for me to be able to handle it atm. Let me know how you would prefer to proceed.

Reply:
In that case just changing back to the old formatting is best

Reply:
ok, I reverse it